WcaOnRails/app/views/persons/_results_by_event.html.erb

```diff
@@ -0,0 +1,61 @@
+<div class="results-by-event">
+  <div class="event-selector text-center">
+    <% @results.map(&:event).uniq.each do |event| %>
```

Comment:
Ah, that is very subtle! I think the clearest implementation would do something like this:

```ruby
@events_competed_in.filter { |e| !e.deprecated? }.each ....
```